src/components/userComponents/SiteMap/SiteMap.tsx

```diff
@@ -71,11 +67,11 @@ function SiteMap({ mode }: SiteMapProps) {
       try {
         let data;
         if (mode === 'tours') {
-          data = await fetchAllSpotlights();
+          data = await fetchSpotlightTours();
```

Comment:
@miha-bhaskaran @sarahhpeng should this be fetching all spotlights?

Reply:
yes